### ISSUE TITLE
TypeError: _log() got an unexpected keyword argument 'exception'

### ISSUE THREAD

**Issue by cfregly** [guildai/guildai#2]
```
ERROR: callback
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/guild/plugins/python_util.py", line 144, in wrapper
    cb(wrapped_bound, *args, **kw)
  File "/usr/local/lib/python2.7/dist-packages/guild/plugins/tensorflow_util.py", line 46, in _handle_summary
    self.log("reading summary values", exception=True)
  File "/usr/local/lib/python2.7/dist-packages/guild/plugin.py", line 73, in log
    log("plugin '%s' %s" % (self.name, msg), *args, **kw)
  File "/usr/lib/python2.7/logging/__init__.py", line 1611, in exception
    error(msg, *args, **kwargs)
  File "/usr/lib/python2.7/logging/__init__.py", line 1603, in error
    root.error(msg, *args, **kwargs)
  File "/usr/lib/python2.7/logging/__init__.py", line 1193, in error
    self._log(ERROR, msg, args, **kwargs)
TypeError: _log() got an unexpected keyword argument 'exception'
```

**Comment by cfregly** [comment]
Using `/guild-examples/mnist2` exactly as instructed in the README.md

**Comment by cfregly** [comment]
this error manifests itself if you don't `pip install psutil` as i've documented in this pull request:

https://github.com/guildai/guild-python/pull/3

**Comment by cfregly** [comment]
@gar1t  is this still an issue?  do we need to run `pip install psutil`?  if not, please close

**Comment by gar1t** [comment]
This has been resolved in 0.3.0.
